estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,106 @@
+                    </div>
+                    <group>
+                        <group>
+<!--                        <field name="state" widget="statusbar" statusbar_visible="open,posted,confirm"/>-->
+                            <field name="state"/>
+                            <field name="property_type_id" options="{'no_create_edit': true, 'no_create': true}"/>
+                            <field name="postcode"/>
+                            <field name="date_availability"/>
+                        </group>
+                        <group>
+                            <field name="expected_price"/>
+                            <field name="best_price"/>
+                            <field name="selling_price"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                            <group>
+                                <field name="description"/>
+                                <field name="bedrooms" string="Bedrooms"/>
+                                <field name="living_area"/>
+                                <field name="facades"/>
+                                <field name="garage"/>
+                                <field name="garden"/>
+                                <field name="garden_area"/>
+                                <field name="garden_orientation"/>
+                                <field name="total_area"/>
+                            </group>
+                            <group>
+                                <field name="state"/>
+                            </group>
+                        </page>
+                        <page string="Offers">
+                            <group>
+                                <field name="offer_ids" />
+                            </group>
+                        </page>
+                        <page string="Other Info">
+                            <group>
+                                <field name="buyer_id" />
+                                <field name="salesperson_id"/>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+     <record id="view_estate_property_search" model="ir.ui.view">
+            <field name="name">estate.property.search</field>
+            <field name="model">estate.property</field>
+            <field name="arch" type="xml">
+                <search string="Estate Property">
+                    <field name="name" string="Title" />
+                    <field name="postcode"/>
+                    <separator/>
+                    <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
+                    <filter string="Available" name="available" domain="['|', ('state', '=', 'new'), ('state' , '=', 'offer received')]"/>
```

Comment:
it's correct; but it's possible to combine these two conditions into one to enhance readibility
```suggestion
                    <filter string="Available" name="available" domain="['('state', 'in', ['new', 'offer received')]"/>
```